pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -336,12 +354,29 @@ def load_triggers(input_file, ifos, vetoes, rw_snr_threshold=None,
                 trigs_dict[path] = dset[above_thresh]
 
             if trigs_dict[path].size == trigs['network/slide_id'][:].size:
-                trigs_dict[path] = slide_filter(trigs, trigs_dict[path],
-                                                slide_id=slide_id)
+                trigs_dict[path] = _slide_filter(trigs, trigs_dict[path],
+                                                 slide_id=slide_id)
 
     return trigs_dict
 
 
+# Wrapper function to load trigger/injection data
```

Comment:
Is it really necessary to introduce such a function? It seems you could just keep using `_load_triggers()`, and just add
```python
if not input_file:
    return None
```
at the very beginning.

Reply:
I blended the two functions together.
The idea of returning None was to avoid that check in all plotting scripts, which may or may not receive an `--inejection-file`.  I don't mind putting it there, i.e.,
```python
if args.injection_file:
    inj_data = load_data(...)
```

Reply:
I see, then I think it makes sense.